tests/Fixer/FunctionNotation/SingleLineThrowFixerTest.php

```diff
@@ -238,10 +238,34 @@ function
         ];
 
         yield [
-            "<?php throw new Exception('a'. 1);",
-            "<?php throw new Exception('a'.
+            "<?php throw new Exception('Message.'. 1);",
+            "<?php throw new Exception('Message.'.
 1
 );",
         ];
+
+        if (\PHP_VERSION_ID >= 70000) {
+            yield [
+                '<?php throw new class() extends Exception { protected $message = "Custom message"; };',
```

Comment:
it can be pretty long
can we, easily, achieve that `throw new class() extends Exception` is enforced to be a single line, and the body of class is formatted properly by `braces` fixer?
if not, let's skill formatting of anonymous classes by `single_line_throw`, imho